src/lib.rs

```diff
@@ -99,6 +99,33 @@ impl ConditionallySelectable for AffinePoint {
     }
 }
 
+impl AffinePoint {
+    /// Get fixed generator point. For speed purposes, we will use a generator of this
+    /// curve that has a small x-coordinate, and its corresponnding y-coordinate.
+    /// The point is then reduced according to the prime field. We need only to
+    /// derive a point with tith a point order = 1 & to check this isn't the identity
+    /// point.
+    ///
+    /// Using: x = 9599346063476877603959045752087700136767736221838581394374215807052943515113
+    ///        y = 2862382881649072392874176093266892593007690675622305830399263887872941817677
```

Comment:
Usually, the numbers are expressed in hex. Just a nitpick, up to you to change it or not.

Reply:
I'm going to keep them as bytes, as the function itself was absent in the library.

Reply:
I was talking about the comments, not the code.

Reply:
Okay, I have changed this.